django/db/backends/sqlite3/schema.py

```diff
@@ -233,7 +233,10 @@ def is_self_referential(f):
         if create_field:
             body[create_field.name] = create_field
             # Choose a default and insert it into the copy map
-            if not create_field.many_to_many and create_field.concrete:
+            if (
+                not (create_field.many_to_many or create_field.computed)
+                and create_field.concrete
+            ):
```

Comment:
When cloning a SQLite database to add a new column, computed columns should not be included or they will cause an error